doc/source/whatsnew/v0.17.1.txt

```diff
@@ -120,3 +120,6 @@ Bug Fixes
 - Bug in ``to_excel`` with openpyxl 2.2+ and merging (:issue:`11408`)
 
 - Bug in ``DataFrame.to_dict()`` produces a ``np.datetime64`` object instead of ``Timestamp`` when only datetime is present in data (:issue:`11327`)
+
+
+- Bug in ``DataFrame.to_csv`` with argument ``date_format`` when dealing with MultiIndexes (:issue:`7791`)
```

Comment:
say something like bug in ``DataFrame.to_csv`` in passing thru arguments for formatting ``MultiIndexes``, including ``date_format``